kubernetes/resource_kubernetes_cron_job_migrate.go

```diff
@@ -0,0 +1,17 @@
+package kubernetes
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceKubernetesCronJobV0() *schema.Resource {
+	schemaV1 := resourceKubernetesCronJobSchemaV1()
+	schemaV0 := patchJobTemplatePodSpecWithResourcesFieldV0(schemaV1)
+	return &schema.Resource{Schema: schemaV0}
+}
+
+func resourceKubernetesCronJobUpgradeV0(ctx context.Context, rawState map[string]interface{}, meta interface{}) (map[string]interface{}, error) {
+	return upgradeJobTemplatePodSpecWithResourcesFieldV0(ctx, rawState, meta)
+}
```

Comment:
How would you feel about adding some debug logging to all the files named `_migrate.go`? Like this one https://github.com/hashicorp/terraform-provider-aws/blob/master/aws/resource_aws_subnet_migrate.go

That would tell us when it finds an empty state for this resource and maybe give us a before/after view of it.

Reply:
Good idea, I'll add in some logging.